packages/purpose-process/src/routers/PurposeRouter.ts

```diff
@@ -78,7 +79,24 @@ const purposeRouter = (
     .post(
       "/reverse/purposes/:id",
       authorizationMiddleware([ADMIN_ROLE]),
-      (_req, res) => res.status(501).send()
+      async (req, res) => {
+        try {
+          const { purpose, isRiskAnalysisValid } =
+            await purposeService.updateReversePurpose({
+              purposeId: unsafeBrandId(req.params.id),
+              reversePurposeUpdateContent: req.body,
+              organizationId: req.ctx.authData.organizationId,
+              correlationId: req.ctx.correlationId,
+            });
+          return res
+            .status(200)
+            .json(purposeToApiPurpose(purpose, isRiskAnalysisValid))
+            .end();
+        } catch (error) {
+          const errorRes = makeApiProblem(error, updatePurposeErrorMapper);
```

Comment:
In this case we use the same ErrorMapper as the updatePurpose because their logic is the same